boards/arm/lpcxpresso55s69/CMakeLists.txt

```diff
@@ -33,4 +36,65 @@ if (CONFIG_BUILD_WITH_TFM)
 	if(${SREC_CAT} STREQUAL SREC_CAT-NOTFOUND)
 		message(FATAL_ERROR "'srec_cat' not found. Please install it, or add it to $PATH.")
 	endif()
+
+	if (CONFIG_TFM_BL2_FALSE)
+		#merge tfm_s and zephyr_ns to a single image
+		#offset needs to be the same value as flash_layout.h in TFM
+		set_property(GLOBAL APPEND PROPERTY extra_post_build_commands
+			COMMAND ${SREC_CAT}
+			ARGS ${CMAKE_BINARY_DIR}/tfm/bin/tfm_s.bin -Binary
+				${CMAKE_BINARY_DIR}/zephyr/${KERNEL_BIN_NAME} -Binary
+				-offset 0x40000
+				-o ${CMAKE_BINARY_DIR}/tfm_merged.bin -Binary
+		)
+	else()
+		#Create and sign for concatenated binary image, should align with the TF-M BL2
+		set_property(GLOBAL APPEND PROPERTY extra_post_build_commands
+
+			#Sign secure binary image with public key
+			COMMAND ${PYTHON_EXECUTABLE} ${TFM_MCUBOOT_DIR}/scripts/wrapper/wrapper.py
+				--layout ${PREPROCESSED_FILE_S}
+				-k ${CONFIG_TFM_KEY_FILE_S}
+				--public-key-format ${TFM_PUBLIC_KEY_FORMAT}
+				--align 1
+				-v ${TFM_IMAGE_VERSION_S}
+				--pad
+				--pad-header
+				${ADD_NS_IMAGE_MIN_VER}
+				-s auto
+				-H 0x400
+				${CMAKE_BINARY_DIR}/tfm/bin/tfm_s.bin
+				${CMAKE_BINARY_DIR}/tfm_s_signed.bin
+
+			#Sign non-secure binary image with public key
+			COMMAND ${PYTHON_EXECUTABLE} ${TFM_MCUBOOT_DIR}/scripts/wrapper/wrapper.py
+				--layout ${PREPROCESSED_FILE_NS}
+				-k ${CONFIG_TFM_KEY_FILE_NS}
+				--public-key-format ${TFM_PUBLIC_KEY_FORMAT}
+				--align 1
+				-v ${TFM_IMAGE_VERSION_NS}
+				-s auto
+				${ADD_S_IMAGE_MIN_VER}
+				-H 0x400
+				${CMAKE_BINARY_DIR}/zephyr/${KERNEL_BIN_NAME}
+				${CMAKE_BINARY_DIR}/zephyr_ns_signed.bin
+
+			#Create concatenated binary image from the two independently signed binary file
+			COMMAND ${PYTHON_EXECUTABLE} ${TFM_MCUBOOT_DIR}/scripts/assemble.py
+				--layout ${PREPROCESSED_FILE_S}
+				-s ${CMAKE_BINARY_DIR}/tfm_s_signed.bin
+				-n ${CMAKE_BINARY_DIR}/zephyr_ns_signed.bin
+				-o ${CMAKE_BINARY_DIR}/tfm_sign.bin
+
+			#Copy mcuboot.bin
+			COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_BINARY_DIR}/tfm/bin/bl2.bin ${CMAKE_BINARY_DIR}/mcuboot.bin
+
+			#Merge mcuboot.bin and tfm_sign.bin for QEMU
```

Comment:
Minor detail, but this chip doesn't support QEMU so just drop the `for QEMU` comment.